pkg/cmd/util/clientcmd/shortcut_restmapper.go

```diff
@@ -121,3 +121,7 @@ func expandResourceShortcut(resource unversioned.GroupVersionResource) unversion
 	}
 	return resource
 }
+
+func GetResourceShortForms() map[string]string {
```

Comment:
This is not used anywhere outside of this package, why do we need to export this otherwise unexported `shortForms` map?